src/main.js

```diff
@@ -3,20 +3,54 @@ import ReactDOM from 'react-dom';
 import _ from 'lodash';
 
 
-const App = () => {
-    const [ score, setScore ] = useState(0);
+function SortableTable({ data }) {
+    const [sortedData, setSortedData] = useState(data);
+    const [clickedColumn, setClickedColumn] = useState(0);
+    const [sortDown, setSortrDown] = useState(true);
+
+
+    const getIsFirstRow = (row, index) => index === 0;
+    const getIsData = (row, index) => index != 0;
+
+    function sortByColumn(columnIndex) {
+        const currentSortDown = (clickedColumn === columnIndex) ? !sortDown : true;
+        const compareFunction = currentSortDown ? (a, b) => (('' + a[columnIndex]).localeCompare(b[columnIndex])) : (a, b) => (('' + b[columnIndex]).localeCompare(a[columnIndex]));
```

Comment:
I like how you handle the comparison, but sometimes creating too much shorthand makes it unreadable.
Can. you break down this line into multiple lines with more elaborate functionality?
Also, NO ONE-LETTER VARIABLES! :)